.flake8

```diff
@@ -1,6 +1,6 @@
 [flake8]
 
-ignore = E731
+ignore = E731, W503
```

Comment:
Could you add comments for the meaning of these codes? Coincidentally I know what W503 is, but I don't know E731 by heart and having to look these up is quite annoying.

Reply:
Done.